app/containers/markdown/index.tsx

```diff
@@ -33,40 +32,32 @@ type UserMention = Pick<IUser, '_id' | 'username' | 'name'>;
 
 interface IMarkdownProps {
 	msg: string;
-	md: MarkdownAST;
-	mentions: UserMention[];
-	getCustomEmoji: Function;
-	baseUrl: string;
-	username: string;
-	tmid: string;
-	isEdited: boolean;
-	numberOfLines: number;
-	customEmojis: boolean;
-	useRealName: boolean;
-	channels: {
+	theme: string;
+	md?: MarkdownAST;
+	mentions?: UserMention[];
+	getCustomEmoji?: Function;
+	baseUrl?: string;
+	username?: string;
+	tmid?: string;
+	isEdited?: boolean;
+	numberOfLines?: number;
+	customEmojis?: boolean;
+	useRealName?: boolean;
+	channels?: {
```

Comment:
Can we create a definition similar to UserMention and apply it on the other places?